snippets/hint-command/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@juananpe/snippet-hint-command",
+  "snippetName": "hint-command",
```

Comment:
We've chatted about this a bit, and since it's currently tied to Groq as a service, we've come to the conclusion that it would make more sense to use `groq` as the snippet name/function name instead – what do you think about that?

Reply:
I have added support for multiple local LLMs (any LLM available locally via Ollama)
If the user doesn't specify a model (optional second parameter), then Groq API will be used.
If the user hasn't provided a Groq API Key, the snippet will display a warning.
Thus, I've renamed the snippet as `llm` (instead of `hint` or `groq`)